From 74232aa068764d70d7b3435c710476eaf8cf3b9b Mon Sep 17 00:00:00 2001
From: "Ye.Li" <Ye.Li@freescale.com>
Date: Fri, 29 May 2015 16:57:16 +0800
Subject: [PATCH] MLK-11001 ARM: dts: Disable the SD1 VSELECT for mx7d 19x19
 lpddr3 arm2 board

The VSELECT for 1.8V does not work on PMIC board, so should not use
it for SD1.

Signed-off-by: Ye.Li <Ye.Li@freescale.com>
---
 arch/arm/boot/dts/imx7d-19x19-lpddr3-arm2.dts | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-19x19-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-19x19-lpddr3-arm2.dts
index 9221272..276b375 100644
--- a/arch/arm/boot/dts/imx7d-19x19-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-19x19-lpddr3-arm2.dts
@@ -102,7 +102,6 @@
 				MX7D_PAD_SD1_CD_B__GPIO5_IO0 		0x59
 				MX7D_PAD_SD1_WP__GPIO5_IO1 			0x59
 				MX7D_PAD_SD1_RESET_B__GPIO5_IO2 	0x59
-				MX7D_PAD_GPIO1_IO08__SD1_VSELECT   	0x59
 
 				MX7D_PAD_GPIO1_IO14__GPIO1_IO14    	0x59
 				MX7D_PAD_GPIO1_IO15__GPIO1_IO15    	0x59
@@ -164,15 +163,13 @@
 };
 
 &usdhc1 {
-	pinctrl-names = "default", "state_100mhz", "state_200mhz";
+	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_usdhc1>;
-	pinctrl-1 = <&pinctrl_usdhc1_100mhz>;
-	pinctrl-2 = <&pinctrl_usdhc1_200mhz>;
 	cd-gpios = <&gpio5 0 0>;
 	wp-gpios = <&gpio5 1 0>;
+	no-1-8-v;
 	keep-power-in-suspend;
 	enable-sdio-wakeup;
-	tuning-step = <2>;
 	vmmc-supply = <&reg_sd1_vmmc>;
 	status = "okay";
 };
-- 
1.8.0

